src/scales/scale.logarithmic.js

```diff
@@ -179,62 +179,94 @@ module.exports = function(Chart) {
 		},
 		getPixelForValue: function(value) {
 			var me = this;
-			var start = me.start;
-			var newVal = +me.getRightValue(value);
-			var opts = me.options;
-			var tickOpts = opts.ticks;
-			var innerDimension, pixel, range;
+			var realValue = +me.getRightValue(value);
+			var reverse = me.options.ticks.reverse;
+			var lg = helpers.log10;
+			var exp = Math.floor(lg(me.minNotZero));
+			var significand = Math.floor(me.minNotZero / Math.pow(10, exp));
+			var firstTickValue = significand * Math.pow(10, exp);
+			var offset = 0;
+			var baseValue, innerDimension, pixel, range, sign;
 
+			if (reverse) {
+				range = {
+					start: me.end,
+					end: me.start
+				};
+				sign = -1;
+			} else {
+				range = {
+					start: me.start,
+					end: me.end
+				};
+				sign = 1;
+			}
+			baseValue = range.start;
 			if (me.isHorizontal()) {
-				range = helpers.log10(me.end) - helpers.log10(start); // todo: if start === 0
-				if (newVal === 0) {
-					pixel = me.left;
-				} else {
-					innerDimension = me.width;
-					pixel = me.left + (innerDimension / range * (helpers.log10(newVal) - helpers.log10(start)));
-				}
+				innerDimension = me.width;
+				pixel = (reverse)
```

Comment:
Probably don't need to wrap `reverse` in parentheses. I also would probably put this on just one line given that it wouldn't be too long of a line

Reply:
IMHO It improves readability.
I don't however want to start a discussion over code style, so if it is a must I will adjust the code as requested.

Reply:
I agree with @benmccann, I would use parenthesis only when it's needed but also I'm not sure useless parenthesis are removed by the minifier.

About the "one line" style, it's not a "must" (else it would be enforced) but most of the code uses the `a ? b : c` form when `a`, `b` and `c` are short enough to make easier to read when on one line (which, I agree, is not necessary perceived the same by everyone).